proseco/guide.py

```diff
@@ -281,11 +281,35 @@ def run_search_stages(self):
         self.log('Done with search stages')
         stage_cands = cand_guides[cand_guides['stage'] != -1]
         stage_cands.sort(['stage', 'mag'])
+        stage_cands = self.exclude_overlaps(stage_cands)
```

Comment:
I went with a strategy of doing this at the end so there was a prioritized list of selectable stars.

Reply:
This still only considers guide stars (not fids or mons).

Reply:
Fids check for stars bright enough to be a guide star with regular spoiler checks. And we can check in sparkles in case that isn't enough. Algorithmically doing everything in proseco does get a bit tricky.